design/release-notes/release-0.9/draft-release-notes.md

```diff
@@ -60,10 +62,10 @@ Users who want to debug from within the cert-manager pod will need to attach an
 additional container with their debug utilities to the pod's namespace.
 
 ### CSRs in Order Resources now PEM Encoded
-CSRs in Order resources have previously been DER encoded however this has now
-changed to PEM encoding. Current orders that were created from a previous
-version of cert-manager will fail to validate and so will be recreated. This
-should resume the order normally.
+CSRs in Order resources have previously been erroneously DER encoded however has
+now been changed to be correctly PEM encoding. Current orders that were created
```

Comment:
`CSRs in Order resources have previously been erroneously DER encoded however has now been changed to be correctly PEM encoding.`

sentence doesn't quite make sense after edits. Needs some new connecting words and maybe change `encoding` to `encoded`? 🤔